zuul.d/ansible-network-jobs.yaml

```diff
@@ -0,0 +1,18 @@
+---
+- job:
+    name: network-ee-integration-tests-network
```

Comment:
we should rename this to network-ee-integration-tests-ansible-network

To be more specific, ideally we do that for vyos too. I'll propose a change later for that